cmd/image-builder/main.go

```diff
@@ -520,12 +520,19 @@ operating systems like Fedora, CentOS and RHEL with easy customizations support.
 
 	rootCmd.AddCommand(describeImgCmd)
 
+	verbose, err := rootCmd.Flags().GetBool("verbose")
+	if err != nil {
+		log.Panicln(err)
+	}
+	if verbose {
+		olog.SetDefault(log.New(os.Stderr, "", log.LstdFlags))
+	}
+
 	return rootCmd.Execute()
 }
 
 func main() {
 	if err := run(); err != nil {
-		fmt.Fprintf(osStderr, "error: %s\n", err)
-		os.Exit(1)
+		log.Fatalf("error: %s\n", err)
```

Comment:
When quickly playing with this it seems by default we now output with data/time prefix:
```console
$ go build && ./image-builder xxx
2025/04/29 17:15:01 error: unknown command "xxx" for "image-builder"
```
I would prefer if we would not do this by default as it adds quite a bit of visual noise.

Reply:
Sure, this is configurable. Let me do that.

Reply:
Fixed:

```
lzap@dev:~/image-builder-cli$ go run ./cmd/image-builder/ xxx 2>/dev/null

lzap@dev:~/image-builder-cli$ go run ./cmd/image-builder/ xxx
error: unknown command "xxx" for "image-builder"
exit status 1
```